### ISSUE TITLE
Multiple copyrights?

### ISSUE THREAD

**Issue by adriendelsalle** [TheSnakePit/quetz#69]
Hi!

I'm not used to copyrights and I noticed multiple files added to the project with a different copyright than the one in the LICENSE file. See [indexing file](https://github.com/TheSnakePit/quetz/blob/master/quetz/indexing.py).

As a contributor, I authored the [cli file](https://github.com/TheSnakePit/quetz/blob/master/quetz/cli.py) but used the `QuantStack` copyright as suggested/asked by the [GitHub license agreement](https://github.com/github/site-policy/blob/1263bf0164cfe60d0900e70ba9126855c7c23c6c/Policies/github-terms-of-service.md#6-contributions-under-repository-license).

@wolfv that's said, do we have to refactor not compliant files?

Thank you!

**Comment by benjamb** [comment]
I'll preface this by declaring IANAL.

It's not uncommon when either contributing significant changes to a file, or creating a new file within a project that uses file-scope copyright notices, that you would be entitled to include your own name (or employer/sponsor). I do agree that this could very quickly become an unmaintainable mess.

+1 for a file at the root of the project declaring authors/contributors.

**Comment by wolfv** [comment]
Yes, I think having a contributors file at the root would be a good addition. So far we have seldomly had too many contributors to a particular file for this to become an issue :) We can always record all contributors into a file later on, and change the copyright to `(c) quetz contributors` or something along those lines.

IMO if you want to add your name feel free! It would be annoying to have this for very simple contributions but whenever you add a file I think you deserve it :)

**Comment by adriendelsalle** [comment]
Great! I totally agree with your feedbacks.

I keep it open to remember us to create a contributor.md, and maybe describe this in the contributing.md file, then refactor copyright headers with a ` (c) quetz contributors` when it will become necessary.

**Comment by SylvainCorlay** [comment]
The copyright model is that every contributor retains the copyright on their contributions

> (c) quetz contributors 

This is what we did for e.g. mamba, listing "Mamba contributors" and "QuantStack" for credit.
